test/test_sparse_csr.py

```diff
@@ -1983,6 +1984,40 @@ def test_exercise_detach(self, device, dtype):
             detached_inp = inp.detach()
             self.assertEqual(inp, detached_inp)
 
+    @skipMeta
+    @all_sparse_compressed_layouts('to_layout')
+    @all_sparse_compressed_layouts('from_layout')
+    def test_compressed_layout_conversions_coverage(self, device, from_layout, to_layout):
```

Comment:
Here only conversions between the sparse compressed layouts are tested. To include the other layouts (coo, strided) into this test, one possibility is to define:
```
def all_sparse_compressed_layouts(test_name='layout', other_layouts=[]):
    return parametrize(test_name, [
        subtest(torch.sparse_csr, name='SparseCSR'),
        subtest(torch.sparse_csc, name='SparseCSC'),
        subtest(torch.sparse_bsr, name='SparseBSR'),
        subtest(torch.sparse_bsc, name='SparseBSC')] + [subtest(layout, name=str(layout)) for layout in other_layouts])
```
and then here have:
```
     @skipMeta
￼    @all_sparse_compressed_layouts('to_layout', other_layouts=[torch.strided, torch.sparse_coo])
￼    @all_sparse_compressed_layouts('from_layout', other_layouts=[torch.strided, torch.sparse_coo])
￼    def test_compressed_layout_conversions_coverage(self, device, from_layout, to_layout):
```

This could be done in a follow-up PR.